src/main/java/com/amplitude/Event.java

```diff
@@ -9,56 +9,162 @@
+   * will be set to the request upload time.
+   */
   public long timestamp = System.currentTimeMillis();
+  /** The current Latitude of the user. */
   public double locationLat;
+  /** The current Longitude of the user. */
   public double locationLng;
 
+  /** The current version of your application. */
   public String appVersion;
+
   public String versionName;
 
+  /** Platform of the device. */
   public String platform;
+  /** The name of the mobile operating system or browser that the user is using. */
   public String osName;
+  /** The device brand that the user is using. */
   public String deviceBrand;
+  /** The device manufacturer that the user is using. */
   public String deviceManufacturer;
+  /** The device model that the user is using. */
   public String deviceModel;
+  /** The carrier that the user is using. */
   public String carrier;
 
+  /** The current country of the user. */
   public String country;
+  /** The current region of the user. */
   public String region;
+  /** The current city of the user. */
   public String city;
+  /** The current Designated Market Area of the user. */
   public String dma;
 
+  /** (iOS) Identifier for Advertiser. */
   public String idfa;
+  /** (iOS) Identifier for Vendor. */
   public String idfv;
+  /** (Android) Google Play Services advertising ID */
   public String adid;
+  /** (Android) Android ID (not the advertising ID) */
   public String androidId;
-
+  /** The (human) language set by the user. */
   public String language;
+  /**
+   * The IP address of the user. Use "$remote" to use the IP address on the upload request. We will
+   * use the IP address to reverse lookup a user's location (city, country, region, and DMA).
+   * Amplitude has the ability to drop the location and IP address from events once it reaches our
+   * servers. You can submit a request to our platform specialist team here to configure this for
+   * you.
+   */
   public String ip;
+  /**
+   * A dictionary of key-value pairs that represent additional data to be sent along with the event.
+   * You can store property values in an array. Date values are transformed into string values.
+   * Object depth may not exceed 40 layers.
+   */
   public JSONObject eventProperties;
+  /**
+   * A dictionary of key-value pairs that represent additional data tied to the user. You can store
+   * property values in an array. Date values are transformed into string values. Object depth may
+   * not exceed 40 layers.
+   */
   public JSONObject userProperties;
 
+  /**
+   * The price of the item purchased. Required for revenue data if the revenue field is not sent.
+   * You can use negative values to indicate refunds.
+   */
   public double price;
+  /** The quantity of the item purchased. Defaults to 1 if not specified. */
   public int quantity;
+  /**
+   * revenue = price quantity. If you send all 3 fields of price, quantity, and revenue, then (price
```

Comment:
`price * quantity`? Something miss in the middle?

Reply:
Good catch. Appears to be a typo on the HTTP V2 API as well. Will alert the team